File: documentation/user/en/solve/examples/filtering-products-in-category/top-selling-products.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		EntityLocaleEquals(CultureInfo.GetCultureInfo("en")),
        		HierarchyWithin(
        			"categories",
        			AttributeEquals("url", "/en/smartwatches")
        		)
        	),
        	OrderBy(
        		AttributeNatural("orderedQuantity", Desc)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("name")
        		),
        		Page(1, 5)
        	)
        )
	)
);

File: documentation/user/en/solve/examples/filtering-products-in-category/top-selling-products.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					entityLocaleEquals(Locale.forLanguageTag("en")),
					hierarchyWithin(
						"categories",
						attributeEquals("url", "/en/smartwatches")
					)
				),
				orderBy(
					attributeNatural("orderedQuantity", DESC)
				),
				require(
					entityFetch(
						attributeContent("name")
					),
					page(1, 5)
				)
			)
		);
	}
);
